### PR TITLE
Align output colors for all tools #130

### LINKED ISSUE #130: Align output colors for all tools
**Now**:
  1. Only for pdtm the color is BrightGreen, while for all the others Green.
  2. Parentheses are colored as well

**Then**:
  1. All the tools versions colored using BrightGreen
  2. Parentheses not colored, just version string and number

### PR DESCRIPTION
**Pull request by edoardottt**
This PR closes #130.

Note: I guess the setup test is failing because of tlsx: 403 Resource not accessible by integration.

### CODE REVIEW

**Review by ehsandeep — APPROVED** [review]

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -97,19 +97,21 @@ func InstalledVersion(tool pkg.Tool, basePath string, au *aurora.Aurora) string
 	if err != nil {
 		osAvailable := isOsAvailable(tool)
 		if !osAvailable {
-			msg = au.Gray(10, "(not supported)").String()
+			msg = "(" + au.Gray(10, "not supported").String() + ")"
```
Comment (ShubhamRasal) [inline pkg/utils/utils.go:100]: we can use fmt.Sprintf to make it more readable
```go
msg = fmt.Sprintf("(%s)", au.Gray(10, "not supported").String())
```

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -97,19 +97,21 @@ func InstalledVersion(tool pkg.Tool, basePath string, au *aurora.Aurora) string
...
 		} else {
-			msg = au.BrightYellow("(not installed)").String()
+			msg = "(" + au.BrightYellow("not installed").String() + ")"
```
Comment (ShubhamRasal) [inline pkg/utils/utils.go:102]:
```go
msg = fmt.Sprintf("(%s)", au.BrightYellow("not installed").String())
```

File: pkg/utils/utils.go [hunk 3]
```diff
@@ -97,19 +97,21 @@ func InstalledVersion(tool pkg.Tool, basePath string, au *aurora.Aurora) string
...
 		}
 	}
 
 	installedVersion := strings.Split(strings.ToLower(outb.String()), "current version: ")
 	if len(installedVersion) == 2 {
 		installedVersionString := strings.TrimPrefix(strings.TrimSpace(string(installedVersion[1])), "v")
 		if strings.Contains(tool.Version, installedVersionString) {
-			msg = au.Green("(latest) (" + tool.Version + ")").String()
+			msg = "(" + au.BrightGreen("latest").String() + ") (" + au.BrightGreen(tool.Version).String() + ")"
```
Comment (ShubhamRasal) [inline pkg/utils/utils.go:110]:
```go
msg = fmt.Sprintf("(%s) (%s)", au.BrightGreen("latest").String(), au.BrightGreen(tool.Version).String())
```

File: pkg/utils/utils.go [hunk 4]
```diff
@@ -97,19 +97,21 @@ func InstalledVersion(tool pkg.Tool, basePath string, au *aurora.Aurora) string
...
 		} else {
-			msg = au.Red("(outdated) ("+installedVersionString+")").String() + " ➡ " + au.Green("("+tool.Version+")").String()
+			msg = "(" + au.Red("outdated").String() + ") (" +
```
Comment (ShubhamRasal) [inline pkg/utils/utils.go:112]:
```go
msg = fmt.Sprintf("(%s) (%s) ➡ (%s)", au.Red("outdated").String(), au.Red(installedVersionString).String(), au.BrightGreen(tool.Version).String())
```

**Review by ShubhamRasal — CHANGES_REQUESTED** [review]
Thanks for raising issue and  PR @edoardottt 
suggesting small change to make it more readable.

**Review by ShubhamRasal — APPROVED** [review]

**Pull request merged**